packages/outputarea/src/data.ts

```diff
@@ -0,0 +1,295 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { nbformat } from '@jupyterlab/coreutils';
+
+import { DatastoreExt } from '@jupyterlab/datastore';
+
+import { IOutputData, OutputData, OutputModel } from '@jupyterlab/rendermime';
+
+import { UUID } from '@phosphor/coreutils';
+
+import {
+  Datastore,
+  Fields,
+  ListField,
+  RegisterField
+} from '@phosphor/datastore';
+
+/**
+ * The namespace for IOutputAreaData interfaces.
+ */
+export namespace IOutputAreaData {
+  /**
+   * An type alias for an output area data schema.
+   */
+  export type Schema = {
+    /**
+     * The id for the schema.
+     */
+    id: string;
+
+    /**
+     * The data fields in the schema.
+     */
+    fields: {
+      /**
+       * Whether the output area is trusted.
+       */
+      readonly trusted: RegisterField<boolean>;
```

Comment:
This field ends up getting unioned with the cell `trusted` field when included with a code cell. I think that's okay, and even kind of nice.